v8.go

```diff
@@ -11,7 +11,7 @@ package v8
 // #include <stdlib.h>
 // #include <string.h>
 // #include "v8_c_bridge.h"
-// #cgo CXXFLAGS: -I${SRCDIR} -I${SRCDIR}/include -std=c++11
```

Comment:
don't remove the two `-I` flags -- they are still needed, it's just that you are getting them from v8_create.go.

Add `-fno-rtti` to v8_create.go as well.

Finally, update the README to include instructions for extracting the ruby gem on osx.